install-streamnzb.sh

```
@@ -0,0 +1,1115 @@
+# Check if installation exists
+check_existing_installation() {
+    if [ -d "$INSTALL_DIR" ] && [ -f "${INSTALL_DIR}/docker-compose.yml" ]; then
+        return 0
+    fi
+    return 1
+}
+
+# Load existing configuration
+load_existing_config() {
+    if [ -f "$CONFIG_FILE" ]; then
+        . "$CONFIG_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)